README.md

```diff
@@ -90,6 +169,51 @@ provided as repository secrets.
 
 [This action](https://github.com/styfle/cancel-workflow-action) is a replacement for the Travis settings **Auto cancel branch builds** and **Auto cancel pull request builds**.
 
+### Triggering a workflow in another repository
```

Comment:
this is a specific use case of a generic action and I am not sure where is the best place to put it, ideas are welcome

Reply:
Indeed

Reply:
at this point, I will move it under `GitHub Actions` paragraph if we want to build a sort of 'common recipes' for GHA

Reply:
Also this is something which is done in few AAE projects only and not a good practice in general as it creates coupling, dependencies across repositories if needed should be mandated to be pull a la dependabot rather than push via direct calls